doc/level0.rst

```diff
@@ -390,10 +394,6 @@ times, such as upon receipt or display. When this happens, the
 Autocrypt state for the sending peer is updated with this new
 information. This update process depends on:
```

Comment:
the update process still depends on the effective date. deleting the effective date here paragraph leaves the list incomplete in regard to this statement.